### PR TITLE
Update ui-testing docs

### PR DESCRIPTION
**Pull request by adinastoll**
# Pull Request Template

## What are you trying to address
Following an engagement where we used Appium (and researched BrowserStack):
- Adding Appium on the list of frameworks that can be used both for Mobile and Web testing. 
- Adding BrowserStack on the list of tools that can be used for Mobile testing.

### CODE REVIEW

**Review by TessFerrandez — APPROVED** [review]
Thank you!

**Comment by adinastoll** [comment]
Hi @TessFerrandez , thank you for the approval! Merging is blocked since the PR requires at least 2 approvals.

File: docs/automated-testing/ui-testing/README.md [hunk 1]
```diff
@@ -61,8 +61,11 @@ Recommended testing frameworks:
   - [Cypress](https://www.cypress.io)
   - [Jest](https://jestjs.io/docs/en/snapshot-testing)
   - [Selenium](https://www.selenium.dev)
+  - [Appium](https://appium.io)
 - OS/Mobile Applications
   - [Coded UI tests (CUITs)](https://learn.microsoft.com/en-us/visualstudio/test/use-ui-automation-to-test-your-code?view=vs-2019)
   - [Xamarin.UITest](https://learn.microsoft.com/en-us/appcenter/test-cloud/uitest/)
+  - [BrowserStack](https://www.browserstack.com)
+  - [Appium](https://appium.io)
```
Comment (tompaana) [inline docs/automated-testing/ui-testing/README.md:69]: Too many Appiums! Lose one :)

Comment (adinastoll) [reply docs/automated-testing/ui-testing/README.md:69]: Hmm, the intention is to add Appium as a valid option for web AND for OS/Mobile (similar to BrowserStack, which works for both). Hence I listed it twice.

Comment (tompaana) [reply docs/automated-testing/ui-testing/README.md:69]: Sorry, my mistake. I didn't notice the nesting.

**Review by tompaana — APPROVED** [review]

**Comment by adinastoll** [comment]
Hey @TessFerrandez I think this PR is ready for merge - but doesn't look like I am allowed to merge myself.

**Pull request merged**